README.md

```diff
@@ -1,9 +1,58 @@
 
-`devtools::install_github("Sage-Bionetworks/dccmonitor")`
\ No newline at end of file
+dccmonitor uses the reticulate package with the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient). See the [reticulate documentation](https://rstudio.github.io/reticulate/#python-version) for more information on setting up reticulate to work with your local Python environment. Additionally, see the [Synapse Python Client](https://github.com/Sage-Bionetworks/synapsePythonClient) for installation instructions.
+
+dccmonitor can be installed via devtools:
+``` R
+devtools::install_github("Sage-Bionetworks/dccmonitor")
+```
+
+## Shiny Application
+
+Using the dccmonitor shiny application requires that the user have a Synapse account, and the user needs access to the specific Synapse folder of interest for monitoring (see Customization for details).
+
+After installing the package, the following will run the application locally:
+
+``` R
+library(dccmonitor)
+shiny::runApp()
+```
+
+**Note:** The application takes time to fully populate the validation information and currently does not have a progress indicator. The application will initialize the study boxes after checking for team membership and gathering the studies represented in the `consortium_fileview` (see Customization for details on this file view). After this, the application will populate the study boxes after the validation checks have run for all represented studies.
+
+## Customization
+
+Customizing the application is done via the config.yml file and the specific file validation checks.
+
+### config.yml
+
+Of the configuration options, only two are specific to dccmonitor while the rest are used to customize the dccvalidator checks (see [Customizing the dccvalidator](https://sage-bionetworks.github.io/dccvalidator/articles/customizing-dccvalidator.html)). The dccmonitor specific configurations are:
+
+- `teams`: Synapse team ID. This is used to verify that the user is a member of the team and, therefore, has access to the folder containing the metadata and documentation files of interest.
+- `consortium_fileview`: Synapse file view ID. A Synapse file view that shows all the files in the `parent` folder (this is the folder that dccvalidator uploads metadata and documentation files to), along with their annotations. The file view should include the columns: id, name, createdOn, createdBy, modifiedOn, currentVersion, study, metadataType, species, assay. Note that the following file-specific annotations are required for dccmonitor to function properly:
+    - documentation: study
+    - manifest: study, metadataType = manifest
+    - individual metadata: study, species, metadataType = individual
+    - biospecimen metadata: study, species, metadataType = biospecimen
+    - assay metadata: study, species, assay, metadataType = assay
+
+Simply set a configuration for the application in the config.yml and change "default" in app.R to your configuration name before running.
```

Comment:
```suggestion
Create a configuration for the application in the config.yml and change "default" in app.R to your configuration name before running.
```

I'd like to avoid "simply", as this might be unfamiliar to some folks and they might not find it simple at first.

Reply:
That's a great point!